defi/src/storeTvlInterval/storeNewTokensValueLocked.ts

```diff
@@ -25,21 +30,28 @@ export default async (
     SK: unixTimestamp,
     ...tvl,
   });
+  const data = { id: protocol.id, timestamp: unixTimestamp, data: tvl }
+  const dayTimestamp = getTimestampAtStartOfDay(unixTimestamp);
+  await Promise.all([
```

Comment:
hmm, maybe put this at the end so if it fails it wont result in failures of the ddb stores, which would be our backup

Reply:
so, I am taking care of this inside the db code, every function is wrapped in try catch, so this should never throw an error. But will move this to the bottom to be safe

Reply:
oke